### PR TITLE
feat: made homepage responsive

### PR DESCRIPTION
**Pull request by abhisekgupta001**
PR: made homepage responsive, please review

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -10,6 +10,61 @@
...
+.hamburger-icon{
+    font-size: 1.5rem;
+    color: var(--white);
+    display: none;
+}
+
+
+/* -----------vertical navigation for mobile---------- */
+.vertical-nav{
+    width: 300px;
```
Comment (znjs) [inline style.css:25]: Suggestion: use relative units for responsiveness

File: style.css [hunk 2]
```diff
@@ -196,7 +251,7 @@
 
 .offer-card-container{
     display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(400px,1fr));
+    grid-template-columns: repeat(auto-fit, minmax(200px,1fr));
     grid-auto-rows: 200px;
```
Comment (znjs) [inline style.css:255]: Suggestion: use relative units for responsiveness

File: style.css [hunk 3]
```diff
@@ -91,7 +146,7 @@
 /* ------------------gender card stylng------------ */
 .gender-cards-container{
     display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
```
Comment (znjs) [inline style.css:149]: Suggestion: use relative units for responsiveness

**Review by znjs — APPROVED** [review]

**Comment by znjs** [comment]
deploy-preview-12--pehnawa netlify app_
Layout is breaking

**Review by muskaanshah — APPROVED** [review]

**Pull request merged**